src/containers/Tenant/Query/QueryResult/i18n/en.json

```diff
@@ -13,6 +13,10 @@
   "title.result": "Result",
   "text_plan-svg": "Execution plan",
   "text_open-new-tab": "Open in new tab",
+  "text_open-new-tab_description": "New tab",
```

Comment:
This description seems to be redundant

Reply:
<img width="276" alt="image" src="https://github.com/user-attachments/assets/ebd61aac-0084-4ac4-b8fb-68977942f33e" />
it is not

Reply:
fixed